CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 CHANGELOG
 =========
 
-0.6.0 (unreleased)
+0.6.0 (2019-15-06)
 ------------------
 
-- Add --safe-write. 0.5.1's behaviour of not allowing overwriting without -f is not the default anymore.
+- Add --safe-write. 0.5.1's behaviour of not allowing overwriting without -f is not the default anymore (breaking change) (#73)
+- Add certificate auth (#61)
+- Remove "requests" backend and the backend argument (breaking change) (#71)
+- Allow to write secret when if 'read' and/or 'list' accesses are not allowed (#70)
```

Comment:
I don't understand the sentence :)

Reply:
When you write a secret, it will try to see if you're overriding an other secret, or if you're trying to write a secret where a folder exists, or if one of the subfolders of your secret is an existing secret. In order to do this, it needs read and list permissions.

When you write a secret but you don't have read or write permission, we don't want it to fail. Your doing something dangerous but you know what you're doing.

(I agree it should be "allow writing" though, and not this "when if", where's this from Oo)

"Write a secret without 'read' and/or 'list' permission"